__tests__/core/runner.test.ts

```diff
@@ -690,4 +690,21 @@ describe('runner', () => {
     const collectOrder = events.map(event => event.type);
     expect(collectOrder).toEqual(realEventsOrder);
   });
+
+  it('run - ensure journey/end is done for real world pages', async () => {
+    const j1 = journey('journey1', async ({ page }) => {
+      step('load homepage', async () => {
+        await page.goto('https://www.elastic.co');
```

Comment:
Its really hard to test this behaviour which is why I added a real world page instead of adding a custom page which contains multiple resources.

If you remove the `minLength: 0`, this test breaks which ensures the fix is working correctly.

Reply:
Can you add this as a comment. My $0.02 is I always try to leave a comment in the code vs. the PR except in the case of comments explaining deleted stuff.

Reply:
Sure will do.